changelog.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.26.0
+
+- New command `state cve open <cve-id>` opens the National Vulnerability Database
+  entry for the given CVE
+- Fixed issue where `state deploy` would fail without a `--path` flag
```

Comment:
These should be in sub-categories, eg. "fixed" and "changed". Please read the document linked above.